docs-src/cloud/action-groups.md

```diff
@@ -0,0 +1,129 @@
+---
+id: action-groups
+title: Action groups
+sidebar_label: Action groups
+description: Action groups permissions for Temporal Cloud and Temporal Server.
+---
+
+Action groups define the permissions and access control for different roles within Temporal Cloud and Temporal Server. They determine which APIs and resources each role (admin, developer, read, write, etc.) can access and interact with.
```

Comment:
```suggestion
Action groups define the permissions and access control for different roles within a Temporal Cloud Namespace. 
They determine which APIs and resources each role (admin, developer, read, write, etc.) can access and interact with.
```

If this is going into Temporal Cloud, then let's drop the reference to the server.